src/components/MessageLogger/consoleList.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from "react";
+import { Message } from "@components/MessageLogger/structs/Message";
+import { MessageList } from "@components/MessageLogger/MessageList";
+
+interface Props {
+    title: string
+    messages: Message[]
+}
+
+export const ConsoleList = ({ title, messages }: Props) => {
+
+    return (
+        <>
+            <h2>{title}</h2>
```

Comment:
aqui hay algo que no cuadra, o no deberia ser un fragment, o no deberia ser un h2, pero no podemos tener multiples h2 en el mismo contenedor

Reply:
es una "bobada" pero es sintomatico de que algo no esta bien planteado

Reply:
I don't understand this